moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp

```diff
@@ -195,6 +197,31 @@ class RobotInterfacePython : protected py_bindings_tools::ROScppInitializer
     return l;
   }
 
+  void setCurrentJointValues(const std::string& name, bp::list& values)
+  {
+    ensureNotMonitoring();
+
+    const robot_model::JointModel* jm = robot_state_->getJointModel(name);
+    if (!jm)
+    {
+      ROS_ERROR_STREAM("joint '" << name << "' does not exist");
+      return;
+    }
+
+    const size_t var_cnt = jm->getVariableCount();
```

Comment:
```suggestion
    const size_t var_count = jm->getVariableCount();
```
avoid excessive abbreviations :-)